src/data/locationHelpers.ts

```diff
@@ -1,4 +1,4 @@
-import { Locations, PrevalenceDataDate } from 'data/location'
+import { Locations } from 'data/location'
```

Comment:
Will importing the json file like this check that types are correct? If not, is there loading we can do to make sure that the data in the file actually matches the PrevalenceData interface?